File: app/src/main/java/com/unsplash/client/data/model/UrlsPhoto.java
package com.unsplash.client.data.model;

import com.google.gson.annotations.SerializedName;

public class UrlsPhoto {

    @SerializedName("regular")
    private String regular;

    @SerializedName("small")
    private String small;

    public String getRegular() {
        return regular;
    }

    public void setRegular(String regular) {
        this.regular = regular;
    }

    public String getSmall() {
        return small;
    }

    public void setSmall(String small) {
        this.small = small;
    }
}

File: app/src/main/java/com/unsplash/client/data/model/UrlsPhoto.kt
package com.unsplash.client.data.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class UrlsPhoto {

    @SerializedName("regular")
    @Expose
    var regular: String? = null

    @SerializedName("small")
    @Expose
    var small: String? = null
}
